docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md

```diff
@@ -0,0 +1,347 @@
+
+### -DisableAutoRepairs
+Turns off the automatic repair of a broken Windows App.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: SetAutoUpdateOptionsSet
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EnableAutomaticBackgroundTask
+Specifies that the automation of updating and repairing will occur as a background task.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: SetAutoUpdateOptionsSet
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ForceUpdateFromAnyVersion
+Specifies that the next update of the Windows App can be of a higher, or lower version.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: SetAutoUpdateOptionsSet
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -HoursBetweenUpdateChecks
+Specifies the time between update checks allowed for a specific Windows App.
+
+```yaml
+Type: UInt32
+Parameter Sets: SetAutoUpdateOptionsSet
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -OptionalPackages
+Specifies the Optional Packages being used by the Windows App.
+
+```yaml
+Type: String[]
+Parameter Sets: SetAutoUpdateOptionsSet
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -RepairUris
+Specifies the location which will be sourced from when repairing the Windows App.
+
+```yaml
+Type: String[]
+Parameter Sets: SetAutoUpdateOptionsSet
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ShowPrompt
+Specifies that if any action is occurring for the Windows App, a prompt will be displayed.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: SetAutoUpdateOptionsSet
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -UpdateBlocksActivation
+Specifies that if an update is available for a Windows App, the App will prevent launching until the update has been installed.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: SetAutoUpdateOptionsSet
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -UpdateUris
+Specifies the location which will be sourced from when updating the Windows App.
+
+```yaml
+Type: String[]
+Parameter Sets: SetAutoUpdateOptionsSet
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -UseSystemPolicySource
+Specifies that an override can be applied to the Developer set settings.
```

Comment:
```suggestion
Specifies that an override can be applied to the Developer Set settings.
```

Reply:
I'm not sure this should be capitalized. Perhaps the word "configured" would be a better fit?

Reply:
Thank you for the comment. I've changed my suggestion to suit.